src/compass/s1_geocode_metadata.py

```diff
@@ -258,6 +258,133 @@ def geocode_calibration_luts(geo_burst_h5, burst, cfg,
         del geocoded_cal_lut_raster
 
 
+
+def geocode_noise_luts(geo_burst_h5, burst, cfg,
```

Comment:
This function has a lot of code repetition with the calibration LUT function. I am wondering whether is possible to consolidate the two in a single function

Reply:
I agree with your concern. The code redundancy has reduced thanks to @LiangJYu.